docs/topics/modules/api_get_component_analyses_response.adoc

```diff
@@ -6,11 +6,9 @@
 
 Fetches various data points for a selected component.
 
-NOTE: You require an {osio} authentication token to invoke this method.
+== Resource URL and access key
 
-== Resource URL
-
-For information about the resource URL, reach out to us on the {osio} support chat channel at link:https://chat.{osio}[https://chat.openshift.io] or email us at link:mailto:openshiftio@redhat.com[openshiftio@redhat.com].
+You require the 3scale gateway URL and access key to invoke all of the analytics API resources mentioned below. For information about the gateway URL and the gateway access key, reach out to us on the fabric8-analytics support chat channel at link:https://chat.openshift.io[https://chat.openshift.io].
```

Comment:
https://chat.openshift.io should be https://chat.openshift.io/developers/channels/fabric8-analytics